appender/src/test/java/com/vlkan/log4j2/redis/appender/RedisServerExtension.java

```diff
@@ -37,7 +37,7 @@ class RedisServerExtension implements BeforeEachCallback, AfterEachCallback {
                     .builder()
                     .port(port)
                     .bind("0.0.0.0")
-                    .setting("user " + username + " on -debug +@all ~* >" + password)
+                    .setting("user " + username + " on -debug +@all ~* &* >" + password)
```

Comment:
I am pretty sure next week I won't recall what these cryptic symbols mean:
```suggestion
                    // Add user, grant access rights, and set password: https://redis.io/docs/latest/operate/oss_and_stack/management/security/acl
                    .setting("user " + username + " on -debug +@all allkeys allchannels >" + password)
```